src/capi/topology.cpp

```diff
@@ -260,7 +255,10 @@ extern "C" chfl_status chfl_topology_bond_order( const CHFL_TOPOLOGY* const topo
     CHECK_POINTER(order);
 
     CHFL_ERROR_CATCH(
-        *order = static_cast<chfl_bond_order>(topology->bond_order(i, j));
+        *order = static_cast<chfl_bond_order>(topology->bond_order(
+            static_cast<size_t>(i),
```

Comment:
Is this for 32-bit support? Should the function take `size_t` as parameter options over `uint64_t`.

Reply:
Yes, this is for 32-bit support. And I think passing uint64_t to the function is better, as this remove a source of variation on the ABI, and make for easier foreign languages bindings.

Reply:
Fair enough just looked odd to me.